tests/java/org/python/core/ReflectedArgsTest.java

```diff
@@ -0,0 +1,53 @@
+package org.python.core;
+
+import junit.framework.TestCase;
+
+/**
+ * Tests for ReflectedArgs.
+ */
+public class ReflectedArgsTest extends TestCase {
+	static class Demo {
+		public static void foo(Object... args) {}
```

Comment:
Please use four spaces for indentation rather than tabs. (I am actually in favor of tabs in general, but one should stick to the project's style in this case)

Reply:
Cheers. I matched the spaces in the other file but didn't think to check the new file.